server/models/project.js

```diff
@@ -52,13 +52,10 @@ projectSchema.set('toJSON', {
 });
 
 projectSchema.pre('save', function generateSlug(next) {
-  const project = this;
-
-  if (!project.slug) {
-    project.slug = slugify(project.name, '_');
+  if (!this.slug) {
+    this.slug = slugify(this.name, { lower: true, strict: true });
   }
-
-  return next();
+  next();
```

Comment:
Does it make a difference that we changed this from `return next()` to just `next()`?